src/network/network-er-x-sfp.md

```diff
@@ -0,0 +1,332 @@
+# EdgeRouter X and EdgeRouter X-SFP
+
+[EdgeRouter X](https://www.ui.com/edgemax/edgerouter-x/) and [EdgeRoute X-SFP](https://www.ui.com/edgemax/edgerouter-x-sfp/) will be both identified in this document as ER-X.  We use version 1.x. 2.x has NOT been tested however preliminary results seem to indicate no immediate issues.
```

Comment:
```suggestion
[EdgeRouter X](https://www.ui.com/edgemax/edgerouter-x/) and [EdgeRoute X-SFP](https://www.ui.com/edgemax/edgerouter-x-sfp/) will be both identified in this document as ER-X.  We use version 1.x. We have not tested version 2.x, however early results indicate no immediate issues.
```

Reply:
want to avoid  "we" and "you" in the documents

Reply:
Maybe simplify it to

```suggestion
[EdgeRouter X](https://www.ui.com/edgemax/edgerouter-x/) and [EdgeRoute X-SFP](https://www.ui.com/edgemax/edgerouter-x-sfp/) will be both identified in this document as ER-X.  Tested on version 1.x of firmware, should work on 2.x as well.
```